include/sys/dataset_kstats.h

```diff
@@ -60,9 +61,15 @@ typedef struct dataset_kstat_values {
 
 typedef struct dataset_kstats {
 	dataset_sum_stats_t dk_sums;
+	zil_sums_t dk_zil_sums;
 	kstat_t *dk_kstats;
 } dataset_kstats_t;
 
+typedef struct dataset_zil_kstat_values_t {
+	dataset_kstat_values_t dkv;
+	zil_stats_t zkv;
+} dataset_zil_kstat_values_t;
```

Comment:
OK for now, but next somebody who add there another member will have to rename it again.